Atomic/client.py

```diff
@@ -41,20 +48,26 @@ def __getattr__(self, name):
 
     def __getattribute__(self, name):
         # Avoid recursion for self._dockerclient
-        if name == "_dockerclient":
+        if name == "_dockerclient" or name == "_client":
             return object.__getattribute__(self, name)
         obj = self._dockerclient
         attr = docker.AutoVersionClient.__getattribute__(obj, name)
         if hasattr(attr, '__call__'):
             def newfunc(*args, **kwargs):
-                result = attr(*args, **kwargs)
-                return iter_subs(result)
+                try:
+                    result = attr(*args, **kwargs)
+                    return iter_subs(result)
+                except requests.exceptions.ConnectionError as e:
```

Comment:
I'm confused why this is necessary if we have the `@property` changes.  Shouldn't we not do the connection until `.dockerclient` is referenced, only reference it in paths that need it?